apps/builder/app/shared/copy-paste/plugin-webflow/__generated__/style-presets.ts

```diff
@@ -0,0 +1,7212 @@
+        type: "unit",
+        unit: "px",
+        value: 20,
+      },
+    },
+    {
+      property: "display",
+      value: {
+        type: "keyword",
+        value: "block",
+      },
+    },
+  ],
+  "w-radio-input": [
+    {
+      property: "float",
+      value: {
+        type: "keyword",
+        value: "left",
+      },
+    },
+    {
+      property: "margin",
+      value: {
+        type: "unparsed",
+        value: "3px 0 0-20px",
+      },
+    },
+    {
+      property: "lineHeight",
+      value: {
+        type: "keyword",
+        value: "normal",
+      },
+    },
+  ],
+  "w-file-upload": [
+    {
+      property: "marginBottom",
+      value: {
+        type: "unit",
+        unit: "px",
+        value: 10,
+      },
+    },
+    {
+      property: "display",
+      value: {
+        type: "keyword",
+        value: "block",
+      },
+    },
+  ],
+  "w-file-upload-input": [
+    {
+      property: "opacity",
+      value: {
+        type: "unit",
+        unit: "number",
+        value: 0,
+      },
+    },
+    {
+      property: "zIndex",
+      value: {
+        type: "unit",
+        unit: "number",
+        value: -100,
+      },
+    },
+    {
+      property: "width",
+      value: {
+        type: "unit",
+        unit: "px",
+        value: 0.1,
+      },
+    },
+    {
+      property: "height",
+      value: {
+        type: "unit",
+        unit: "px",
+        value: 0.1,
+      },
+    },
+    {
+      property: "position",
+      value: {
+        type: "keyword",
+        value: "absolute",
+      },
+    },
+    {
+      property: "overflow",
+      value: {
+        type: "keyword",
+        value: "hidden",
+      },
+    },
+  ],
+  "w-file-upload-default": [
+    {
+      property: "color",
+      value: {
+        type: "rgb",
+        alpha: 1,
+        r: 51,
+        g: 51,
+        b: 51,
+      },
+    },
+    {
+      property: "display",
+      value: {
+        type: "keyword",
+        value: "inline-block",
+      },
+    },
+  ],
+  "w-file-upload-uploading": [
+    {
+      property: "color",
+      value: {
+        type: "rgb",
+        alpha: 1,
+        r: 51,
+        g: 51,
+        b: 51,
+      },
+    },
+    {
+      property: "display",
+      value: {
+        type: "keyword",
+        value: "inline-block",
+      },
+    },
+  ],
+  "w-file-upload-success": [
+    {
+      property: "color",
+      value: {
+        type: "rgb",
+        alpha: 1,
+        r: 51,
+        g: 51,
+        b: 51,
+      },
+    },
+    {
+      property: "display",
+      value: {
+        type: "keyword",
+        value: "inline-block",
+      },
+    },
+  ],
+  "w-file-upload-error": [
+    {
+      property: "marginTop",
+      value: {
+        type: "unit",
+        unit: "px",
+        value: 10,
+      },
+    },
+    {
+      property: "display",
+      value: {
+        type: "keyword",
+        value: "block",
+      },
+    },
+  ],
+  "w-file-upload-uploading-btn": [
```

Comment:
what are these? Do not look like tags

Reply:
the keys are either tags or classes, I will start using them in the next prs

Reply:
its how webflow defines styles on their custom abstractions like a link button etc

Reply:
they are not copied to clipboard too?